snippets/base/admin/fields.py

```diff
@@ -20,9 +20,11 @@ def clean(self, value):
 class JEXLChoiceField(ChoiceField):
     def __init__(self, attr_name, *args, **kwargs):
         self.attr_name = attr_name
+        self.jexl = '{attr_name} == {value}'
+        self.jexl = kwargs.pop('jexl', self.jexl)
```

Comment:
I agree with making things as clear as possible so they can be maintainable in the future.

The in thing with this specific one is the actually `to_jexl` does *not* require `attr_name` or `value`. It does provide them as `format()` arguments but the string doesn't have to use them. In fact in `profileAgeCreated` I didn't use them ([code](browserSettings.update.autoDownload)), so the JEXL expr is easier to read and more maintainable.

E.g. instead of being
   `'(currentDate - {attr_name}) / 604800000) >= {value}',` 
 I commited it as
   `'((currentDate - profileAgeCreated) / 604800000) >= {value}',`

If you insist I don't mind changing this to use `attr_name` and add a check but I prefer it this way